app/src/main/res/values/strings.xml

```diff
@@ -163,8 +163,8 @@
     <string name="right_to_left_rendering__abbreviated">RTL rendering</string>
     <string name="highlight_line_ending">Highlight line ending</string>
     <string name="highlight_line_ending_if_two_or_more_spaces">Highlight spaces at the end of lines containing two or more spaces</string>
-    <string name="use_monospace_for_code">Use Monospace for code</string>
-    <string name="use_different_fonttype_slow_down">Make use of a different font type. Changing the font type dynamically makes an impact on the performance of the app.</string>
+    <string name="use_monospace_for_code">Use monospace font for code</string>
```

Comment:
Assuming here that the font isn't called "Monospace"

Reply:
correct